crates/astria-conductor/src/executor.rs

```diff
@@ -268,49 +355,21 @@ impl<C: ExecutionClientV1Alpha1> Executor<C> {
                 // execute_block will check if our namespace has txs; if so, it'll return the
                 // resulting execution block hash, otherwise None
                 let Some(execution_block_hash) = self
-                    .execute_block(block)
+                    .execute_block(block.clone())
                     .await
                     .wrap_err("failed to execute block")?
                 else {
                     // no txs for our namespace, nothing to do
-                    debug!("execute_block returned None; skipping finalize_block");
+                    debug!("execute_block returned None; skipping call_update_commitment_state");
                     return Ok(());
                 };
 
-                // finalize the block after it's been executed
-                self.finalize_block(execution_block_hash, sequencer_block_hash)
+                self.update_commitment_state_firm(block, execution_block_hash)
```

Comment:
We need to update the full commitment here. Execution no longer updates the soft commitment, it's a separate action on a different resource.